rkt/fetch.go

```diff
@@ -17,24 +17,21 @@ package main
 import (
 	"fmt"
 	"io"
-	"io/ioutil"
 	"net/http"
 	"net/url"
 	"os"
-	"path/filepath"
 	"runtime"
 	"strings"
 	"time"
 
-	"github.com/coreos/rocket/cas"
-	"github.com/coreos/rocket/pkg/keystore"
-
-	"github.com/coreos/rocket/Godeps/_workspace/src/github.com/appc/docker2aci/lib"
+	docker2aci "github.com/coreos/rocket/Godeps/_workspace/src/github.com/appc/docker2aci/lib"
```

Comment:
I'm not actually sure what happened here, goimports kept trying to remove the import until I pinned the name